code/frontend/packages/ranking-client/src/app/_shared/models/game.model.ts

```diff
@@ -28,6 +30,7 @@ export class Game {
     this.set3Team1 = args.set3Team1;
     this.set3Team2 = args.set3Team2;
     this.winner = args.winner;
+    this.rankingPoints = args.rankingPoints;

```

Comment:
this probably should be mapped to the ranking point model:

`args.rankingPoints?.map(r => new RankingPoint(r))`